src/components/shared/ListSelect/ListSelect.js

```diff
@@ -14,8 +15,12 @@ export const ListSelect = memo(({
   onChange
 }) => {
   const intl = useIntl();
+  const stripes = useStripes();
+  const isCentral = stripes.user?.user?.consortium?.centralTenantId;
 
-  const options = identifierOptions[capabilities]?.map((el) => ({
+  const options = identifierOptions[capabilities]?.filter((el) => {
```

Comment:
`filter` and `map` can be replaced with `reduce`:
```
const options = identifierOptions[capabilities]?.reduce((accum, cur) => {
  if (isCentral && cur.isCentralTenant) {
    return accum;
  }
  
  const {value, label, disabled} = cur;
  
  accum.push({
    value,
    label: intl.formatMessage({ id: label }),
    disabled
  });
  
  return accum;
}, []);
```

Reply:
True, @SKarolFolio , but personally I like the clarity of filter/map over the efficiency of reduce. If this list had 1000s of entries, or the filter/map operations were expensive, then it would be important to reduce iterations but that's not a significant concern here. Your implementation is fine though, and so is the original :) 

It is great to suggest alternatives approaches but I worry about blocking approval based on a personal preference for equivalent implementations.

Reply:
Hi @SKarolFolio I agree that `reduce` will work faster, but `filter` with `map` more easy to understand what is happening here.